spec/views/layouts/application.html.slim_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   before do
     allow(view).to receive(:user_fully_authenticated?).and_return(true)
+    allow(view).to receive(:current_user).and_return(User.new)
```

Comment:
I get the following error when running the specs without that line:

```
span.bold = current_user.email
     
     ActionView::Template::Error:
       undefined method `email' for nil:NilClass
```

The `span.bold` tag in question is in the `_nav_auth` partial. Adding an explicit current_user obviously fixes it, although I'm not sure if it points to a deeper issue :slightly_frowning_face: